.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -1,3 +1,9 @@
+---
+name: Bug report
```

Comment:
Since everything else in the other template names is capitalized, I'd do "Report` here.